lib/components/clock/clock.jsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+
+import * as actions from '../../actions/clock';
+
+export class Clock extends React.Component {
+    componentDidMount() {
+        this.props.getTime();
```

Comment:
I disagree with this, there is still a flow of component -> action creator -> store and so forth. This is also a good show of immutability, where two clock components will both try and get an update/initial time but they won't necessarily re-render. Also, having the update and get is just to showcase two different actions.